src/js/components/FilterCriteria.jsx

```diff
@@ -91,11 +94,32 @@ class FilterCriteria extends Component {
                 onChange={(ev) => onChangeValue(i, ev.target.value)}
               />
               {(() => {
+                if (filters.length !== 1) {
+                  return (
+                    <FloatingActionButton
+                      mini={true}
+                      primary={true}
+                      style={{
+                        ...style,
+                        margin: '1em 0 0 1em'
+                      }}
+                      onTouchTap={(ev) => onRemove(i, ev)}
```

Comment:
I think you should keep `ev` as the first parameter so that it is similar to a regular `onTouchTap` event handler